webpack.config.js

```diff
@@ -76,6 +76,10 @@ module.exports = {
         ? "[chunkhash].bundle.js"
         : "bundle.js",
   },
+  // https://github.com/mapbox/mapbox-gl-draw/issues/626
+  node: {
```

Comment:
I'm not 100% on this. Without this fix the app won't compile with the `mapbox-gl-draw` included, but I don't complete understand what the issue is and why this wepack config line fixes it.

Reply:
Another alternative that works which I think might be preferable to the options above is to just import the built js file directly:
```
import MapboxDraw from "@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw";
```
This avoids the need to make any webpack config changes.